statsmodels/graphics/tests/test_functional.py

```diff
@@ -17,13 +17,20 @@
     have_matplotlib = False
 
 
-data = elnino.load(as_pandas=False)
-labels = data.raw_data[:, 0].astype(int)
-data = data.raw_data[:, 1:]
+@pytest.fixture()
```

Comment:
Will these be useful elsewhere?  i.e. would it be worthwhile to put them in a conftest?

Reply:
I these are specific enough t this module to not warrant ending up in conftest.